garden_ai/pipelines.py

```diff
@@ -161,7 +162,7 @@ def _collect_requirements(self):
 
         self.python_version = py_versions["pipeline"] or py_versions["system"]
         self.conda_dependencies = list(set(self.conda_dependencies))
-        self.pip_dependencies = validate_pip_lines(list(set(self.pip_dependencies)))
+        self.pip_dependencies = list(set(validate_pip_lines(self.pip_dependencies)))
```

Comment:
Was something breaking or did you just notice dupes and wanted to be proactive?

Reply:
yeah the latter, just noticed it was duplicating dependencies and wanted to be proactive